src/main/java/org/example/examplefinalProject/entity/ClassRoom.java

```diff
@@ -13,7 +12,7 @@ public class ClassRoom {
     private String classRoomName;
 
     @OneToOne(optional = false, fetch = FetchType.EAGER)
-    @JoinColumn(name = "teacher_id")
+    @JoinColumn(name = "teacherId")
```

Comment:
why you changed this? if so, dont forget to update database using flyway

Reply:
flyway ještě není nasazen